client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -188,6 +190,17 @@ const JetpackComMasterbar: React.FC< Props > = ( { pathname } ) => {
 						>
 							<JetpackLogo full size={ 38 } />
 						</ExternalLink>
+						{ shouldShowCart && (
+							<MasterbarCartWrapper
+								goToCheckout={ goToCheckout }
+								selectedSiteSlug={ siteSlug || undefined }
+								selectedSiteId={ siteId || undefined }
+								checkoutLabel={ translate( 'Go to Checkout' ) }
+								cartIcon={ CloudCartIcon }
+								emptyCart={ EmptyCart }
```

Comment:
Let me know if I'm wrong, but since passing element types is uncommon in React, would it work to change the props here to accept a component instead? To my knowledge the only reason to pass an element is if you want to avoid rendering the component until the last minute (eg: if it needs additional props) but `EmptyCart` seems relatively simple.

```suggestion
								emptyCart={ <EmptyCart /> }
```